python2.7/Dockerfile

```diff
@@ -0,0 +1,42 @@
+ARG ARCH=""
+ARG ALPINE_VERSION="3.12"
+
+FROM alpine:${ALPINE_VERSION}
+
+ARG PYINSTALLER_TAG
+ENV PYINSTALLER_TAG ${PYINSTALLER_TAG:-"v3.6"}
+
+# Official Python base image is needed or some applications will segfault.
```

Comment:
Yes, have used alpine3.12 and install python in it. because the image is not available with the alpine 3.12 image and python 2.7.
The last alpine version available with python 2.7 is alpine 3.11.5.

Reply:
The comment is wrong then. It's not using the official Python base image.

Reply:
python2.7 official image is not available So have used it like this.. 
So shall i use python2.7 image with alpine3.11. is that fine?